apps/opencs/view/render/scenewidget.cpp

```diff
@@ -140,59 +197,6 @@ void RenderWidget::toggleRenderStats()
     window->getEventQueue()->keyRelease(osgGA::GUIEventAdapter::KEY_S);
 }
 
-
-// --------------------------------------------------
-
-CompositeViewer::CompositeViewer()
-    : mSimulationTime(0.0)
-{
-    // TODO: Upgrade osgQt to support osgViewer::ViewerBase::DrawThreadPerContext
-    // https://gitlab.com/OpenMW/openmw/-/issues/5481
-    setThreadingModel(osgViewer::ViewerBase::SingleThreaded);
-
-#if OSG_VERSION_GREATER_OR_EQUAL(3,5,5)
-    setUseConfigureAffinity(false);
```

Comment:
This line was important, so don't forget it.

Reply:
Right. I added that to CompositeOsgRenderer in commit b10a551 . Still left out the `OSG_VERSION_GREATER_OR_EQUAL` because using that would require some extra stuff in.. umm. cMakeLists maybe? Anyway, perhaps it's better to do the check in SceneWidget, then set a bool to CompositeOsgRenderer to control whether that is set or not.

Reply:
The function doesn't exist in older OSG versions, so definitely keep the check until we officially drop support for anything older. The header with those macros should be included (potentially through like forty levels of indirection) in basically any OSG header, though, so I don't see why you'd think you need CMake changes.

Reply:
Ah, so not CMake, but I need to add a library, as it doesn't compile if I just add that to CompositeOsgRenderer. It's strangely not in any .h or .hpp, but in src/osg/Version.in, anyway, I'll add that check somehow somewhere, either to CompositeOsgRenderer or to SceneWidget.

Reply:
For me, it's in `<osg install location>/include/osg/Version`. OSG headers don't have a file extension, so maybe that's what was tripping you up. You shouldn't need to meddle with anything at all as the file's already got `#include <osg/Version>` at the top.